buda/entities/signup.proto

```diff
@@ -11,6 +11,22 @@ message Signup {
     Uuid user_id = 2;
     Uuid visitor_id = 3;
     Uuid legacy_visitor_id = 100; //buffer-web visitor_id, will be deprecated
-
+    Uuid client_id = 5;
     google.protobuf.Timestamp created_at = 4;
+    enum SignupOption {
+      EMAIL = 0;
+      FACEBOOK = 1;
+      TWITTER = 2;
+      LINKEDIN = 3;
+      GOOGLE = 4;
+      ORG_TEAM_INVITE = 5;
+    }
+    SignupOption signup_option = 6;
+    bool with_trial = 7;
+    enum Product {
```

Comment:
One tiny tweak I would make would be to move the enum declarations to the top of the message above the fields, like we do with other nested types, for example [here](https://github.com/bufferapp/buda-protobufs/blob/master/buda/entities/subscription_created.proto#L13-L32). This way it's easy to see all the fields together and all the nested declarations in one place. How does that feel @mallen5311?

Reply:
Feels great 🙌 , thanks for the reviews!  Haha, taking a bit to remember some of this from when I wrote protobufs over the summer 😅 .

Reply:
I just committed changes, feel ok to merge?

Reply:
@mallen5311 everything feels good, except I'm still unsure about the `client_id` field. Curious if for now we can instead use an enum there instead to have it look like the ideal field, perhaps something like this?

```enum SignupClient {
    WEB = 0;
    APPLE = 1; //Or maybe IOS, not sure
    ANDROID = 2;
    OTHER = 3;
}
```

Reply:
Cool, yeah, I on board with that 👍 .  I just committed this change, and went with the IOS naming convention since we have used that in other places.